src/Workspaces/Core/Portable/Formatting/TriviaEngine/AbstractTriviaFormatter.cs

```diff
@@ -818,7 +818,7 @@ private string GetWhitespaceString(LineColumn lineColumn, LineColumnDelta delta)
             var useTabOnlyForIndentation = this.OptionSet.GetOption(FormattingOptions.UseTabOnlyForIndentation, this.Language);
```

Comment:
Ooh, for what it's worth, that option sounds awesome. But if I had to choose, I'd rather have #1151 fixed than the option (for now).